### PR TITLE
Revised setting headline

### PR DESCRIPTION
**Pull request by tonymurphy-rgb**
Shortened the copy to be more like other settings headlines.

### CODE REVIEW

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://circleci.com/gh/mozilla-services/guardian-vpn-windows/539/artifacts/0/Users/circleci/project/test/result/Output.log' target='_blank'>details</a></strong>

File: ui/src/UI/Resources/Localization/Translations/en-us.ftl [hunk 1]
```diff
@@ -81,7 +81,7 @@ tray-menu-hide = _Hide
 settings-auto-launch = Launch VPN app on computer startup
 settings-unsecured-network-title = Unsecured network alert
 settings-unsecured-network-content = Get notified if you connect to an unsecured Wi-Fi network
-settings-local-device-access-title = Allow access to your local network
+settings-local-device-access-title = Local network access
```
Comment (connected-mhuang) [inline ui/src/UI/Resources/Localization/Translations/en-us.ftl]: This might fail the UI test. Please update the same string in UI tests.

Comment (connected-nkozul) [reply ui/src/UI/Resources/Localization/Translations/en-us.ftl]: @connected-mhuang Could you point them in the right direction?

Comment (connected-mhuang) [reply ui/src/UI/Resources/Localization/Translations/en-us.ftl]: @tonymurphy-rgb Could you please update this line as well? https://github.com/mozilla-services/guardian-vpn-windows/blob/680ff1cff71b2dc5b8927bd5e5241ba1b32b2551/test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Tests/SettingTest.cs#L89

**Comment by connected-nkozul** [comment]
Superseded by #78 .

**Pull request closed without merging**